src/Components/Style.css

```diff
@@ -0,0 +1,73 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  /* min-width: 1200px; */
```

Comment:
**You have done a good job!**

- [ ] Your app has some cleanup jobs to complete. For starting, please remove this commented code!